perfkitbenchmarker/managed_relational_db.py

```diff
@@ -0,0 +1,107 @@
+from abc import ABCMeta, abstractmethod
+import uuid
+
+from perfkitbenchmarker import flags
+from perfkitbenchmarker import resource
+
+# TODO (ferneyhough): change to enum
+flags.DEFINE_string('database', None,
+                    'Managed database flavor to use (mysql, postgres)')
+flags.DEFINE_string('database_name', None,
```

Comment:
And ill be sure to watch out for that. I don't think it is appropriate for this class to have any notion of benchmark-specific requirements, however.